content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -1,7 +1,1704 @@
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-pp PROJECT_PATH, --project-path PROJECT_PATH`**
+
+  The path to the project.
+
+- **`-pn PROJECT_NAME, --project-name PROJECT_NAME`**
+
+  The name of the project.
+
+- **`-gp GEM_PATH, --gem-path GEM_PATH`**
+
+  The path to the Gem.
+
+- **`-gn GEM_NAME, --gem-name GEM_NAME`**
+
+  The name of the Gem.
+
+- **`-egf ENABLED_GEM_FILE, --enabled-gem-file ENABLED_GEM_FILE`**
+
+  The CMake file that manages the list of Gems that are enabled. When disabling a Gem, this command removes the Gem from the specified file. If not specified, this command uses the `Code/enabled_gem.cmake` file.
+
+- **`-ohf OVERRIDE_HOME_FOLDER, --override-home-folder OVERRIDE_HOME_FOLDER`**
+
+  Overrides the default home folder with the specified folder. By default, the home folder is the user folder. 
+
+
+<!-------------------------------------------------------------->
+
+## `edit-engine-properties`
+
+Edits the specified engine's properties by modifying the `engine.json` file. 
+
+### Format
+
+```cmd
+edit-engine-properties [-h] (-ep ENGINE_PATH | -en ENGINE_NAME)
+                        [-enn ENGINE_NEW_NAME]
+                        [-ev ENGINE_VERSION]
+```
+
+### Usage
+
+Updates the `engine_name` field in the `engine.json` file located at the specified engine's folder. 
+
+```cmd
+o3de.bat edit-engine-properties --engine-path ENGINE_PATH --engine-new-name ENGINE_NEW_NAME
+
+// or
+
+o3de.bat edit-engine-properties --engine-name ENGINE_NAME --engine-new-name ENGINE_NEW_NAME
+```
+
+
+### Optional parameters
+
+  
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-ep ENGINE_PATH, --engine-path ENGINE_PATH`**
+
+  The path to the engine.
+
+- **`-en ENGINE_NAME, --engine-name ENGINE_NAME`**
+
+  The name of the engine.
+
+
+**Engine properties:**
+
+The following parameters modify the specified engine's properties.
+  
+- **`-enn ENGINE_NEW_NAME, --engine-new-name ENGINE_NEW_NAME`**
+
+  Sets the engine's name. 
+
+- **`-ev ENGINE_VERSION, --engine-version ENGINE_VERSION`**
+
+  Sets the engine's version.
+
+
+<!-------------------------------------------------------------->
+
+## `edit-project-properties`
+
+Edits the specified project's properties by modifying the `project.json` file. 
+
+### Format 
+
+```cmd
+edit-project-properties [-h]
+                        (-pp PROJECT_PATH | -pn PROJECT_NAME)
+                        [-pnn PROJECT_NEW_NAME]
+                        [-po PROJECT_ORIGIN]
+                        [-pd PROJECT_DISPLAY]
+                        [-ps PROJECT_SUMMARY]
+                        [-pi PROJECT_ICON]
+                        [-at [ADD_TAGS [ADD_TAGS ...]] | -dt
+                        [DELETE_TAGS [DELETE_TAGS ...]] | -rt
+                        [REPLACE_TAGS [REPLACE_TAGS ...]]]
+```
+
+### Usage
+
+Updates the "project_name" field within the gem.json located at the supplied project path or at the path of the registered project with <project-name>
```

Comment:
```suggestion
Updates the `project_name` field in the `gem.json` file located in the supplied project path or at the path of the registered project with <project-name>.
```
Does "with <project-name>" modify "Updates the `project_name` field? If so, consider this rewrite for flow/clarity:

>Updates the `project_name` field with <project-name> in the `gem.json` file located in the supplied project path or at the path of the registered project.

However, if "registered project with <project-name>" reads as intended, disregard this additional suggestion.